src/OrleansSQLUtils/CreateOrleansTables_SqlServer.sql

```diff
@@ -106,8 +105,7 @@ BEGIN
 	(
 		[DeploymentId] NVARCHAR(150) NOT NULL, 
 		[Timestamp] DATETIME2(3) NOT NULL DEFAULT GETUTCDATE(), 
-		[Version] BIGINT NOT NULL,		
-		[ETag] ROWVERSION NOT NULL,
+		[Version] BIGINT NOT NULL DEFAULT 0,
```

Comment:
Shay, can you please also update the comments in the code? It now says: 
"-- the ETag is ROWVersion in SQL Server 2005 and later whereas in SQL Server 2000 UNIQUEIDENTIFIER is used"